src/AvtaleSide/steg/KontaktInformasjonSteg/kontorInfo/deltakerInfo.less

```diff
@@ -1,5 +1,5 @@
 .deltakerinfo {
-    margin: 0 0.5rem 2.5rem 0.5rem;
+    margin: 0.5rem 0.5rem 2.5rem 0.5rem;
```

Comment:
Det er kanskje bedre å legge på margin på `HemmeligAdresseVarsel` boksen enn å endre containeren. Denne endringen blir jo gjeldende selv om `HemmeligAdresseVarsel` ikke vises og da blir det kanskje plutselig litt for mye margin?